packages/sqflite/tizen/src/database_manager.h

```diff
@@ -0,0 +1,65 @@
+#ifndef DATABASE_MANAGER_H_
+#define DATABASE_MANAGER_H_
+
+#include <flutter/standard_method_codec.h>
+
+#include "list"
+#include "sqlite3.h"
+#include "string"
+
+struct DatabaseError : public std::runtime_error {
+  DatabaseError(int code, const char *msg)
+      : std::runtime_error(std::string(msg) + " (code " + std::to_string(code) +
+                           ")") {}
+};
+
+class DatabaseManager {
+  static const int BUSY_TIMEOUT_MS = 2500;
+
+ public:
+  sqlite3 *sqliteDatabase;
+  std::map<std::string, sqlite3_stmt *> stmtCache;
+  bool singleInstance;
+  std::string path;
+  int id;
+  int logLevel;
```

Comment:
see : 
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules
https://google.github.io/styleguide/cppguide.html#Variable_Names
https://google.github.io/styleguide/cppguide.html#Class_Format 

Please add underscore to the class member names.

Why are these member variables `public`?